testing/vcs/test_vcs_set_cmap_colorcell.py

```diff
@@ -0,0 +1,10 @@
+import vcs
```

Comment:
Instead of making two tests.. can you just make one? We have lot many tests and now we need to make sure that we don't grow the number as much